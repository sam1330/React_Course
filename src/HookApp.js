import React from "react";

export const HookApp = () => {
  return (
    <>
      <h1 class="bg-dark">Hola Mundo</h1>
    </>
  )
};

export default HookApp;