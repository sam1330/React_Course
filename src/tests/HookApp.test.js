import React from "react";
import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe('<HookApp /> Tests', () => {

  test('Should show correctly', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
  
});