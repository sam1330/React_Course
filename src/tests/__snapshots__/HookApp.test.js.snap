// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HookApp /> Tests Should show correctly 1`] = `
<Fragment>
  <h1
    class="bg-dark"
  >
    Hola Mundo
  </h1>
</Fragment>
`;
